packages/runtime/src/widgets/Charts/index.tsx

```diff
@@ -59,13 +65,19 @@ const tabsConfig = {
 };
 
 export const Chart: React.FC<ChartProps> = (props) => {
-  const { type } = props;
+  // eslint-disable-next-line @typescript-eslint/no-implied-eval, no-new-func
+  const config = new Function(
+    `return ${props.config?.toString() || "{}"}`,
+  )() as ChartConfigs;
```

Comment:
You can just serialize the config directly no need to wrap as a separate function

Reply:
Cannot use `JSON.parse` directly as it has some limitations such as https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/JSON_bad_parse#property_names_must_be_double-quoted_strings